### PR TITLE
test

### PR DESCRIPTION
**Pull request by Wvarquez**
test

### CODE REVIEW

**Comment by Shandenr** [comment]
Test

**Comment by Jerramel** [comment]
test

**Comment by kparilla** [comment]
test

**Review by kparilla — COMMENTED** [review]
Change line zodiacSign(1,7) to  zodiacSign(3,10)

File: index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 const zodiacSign = require('get-zodiac-sign');
-var myZodiacSign1 = zodiacSign(1,7);
-var myZodiacSign2 = zodiacSign(12,1);
+var myZodiacSign1 = zodiacSign(3,13);
+var myZodiacSign2 = zodiacSign(2,18);
```
Comment (Jerramel) [inline index.js:3]: Change myZodiacSign2 = zodiacSign(2,18) to myZodiacSign2 = zodiacSign(2,16)

**Comment by debasisb** [comment]
Closing this pull request

**Pull request closed without merging**